py3status/command.py

```diff
@@ -83,16 +86,33 @@ def click(self, data):
             if module['type'] == 'py3status':
                 name = module['module'].module_name
                 instance = module['module'].module_inst
+
+                # this is a frame click request
+                if name == 'frame':
+                    # if the 'frameclick' named click event was requested
+                    # we simulate a click on the frame button instead
+                    if button == 0:
+                        button = 1
+                        instance = '{} button'.format(
+                            ''.join(module_name.split(' ')[1:]))
```

Comment:
Something to mention. Users won't always add `instance`, eg `order += "frame"` would give us `instance = ' button'` (with a leading space). Still work tho. :zap: